main.py

```diff
@@ -1,125 +1,118 @@
 #################
 #### Imports ####
 #################
-
+import os.path
 from datetime import datetime
+from pathlib import Path
+
+from ba_infra.presto_connection import WixPrestoConnection
+from google.oauth2 import service_account
 
 import functions as func
 
-###########################
-#### Global Parameters ####
-###########################
+#####################
+####  Parameters ####
+#####################
+DATA_FOLDER = Path(os.getcwd())
 
-api_key = # Upon production the key will be taken from airflow storage, during development the key is taken from text file
+BQ_CREDENTIALS = service_account.Credentials.from_service_account_file(DATA_FOLDER / "key.json", scopes=["https://www.googleapis.com/auth/cloud-platform"])
+API_KEY = func.read_json(path_file=DATA_FOLDER / "key_api.json")['api_key']
+
+GLOBAL_PARAMETERS = func.read_json(path_file=DATA_FOLDER / "configuration.json")
 
 # Set dates
-date_mid_month = func.mid_month()   # Data updated two weeks after the end of the month, for the previous month
-current_time = datetime.now()
+DATE_MID_MONTH = func.mid_month()   # Data updated two weeks after the end of the month, for the previous month
+CURRENT_TIME = datetime.now()
+
+# Set Tables
+GOOGLE_SHEET_KW_TABLE = 'gdrive.bi.marketing_itaib_semrush_kw_gs'

```

Comment:
add these constants to the config file:
```
'tables': {'kw': 
               {'google_sheet' : 'gdrive.bi.marketing_itaib_semrush_kw_gs', 'table': 'sandbox.itaib.seo_kw_semrush'},
               'url':
               {'google_sheet' : 'gdrive.bi.marketing_itaib_semrush_url_gs', 'table': 'sandbox.itaib.seo_url_semrush'},
               ...}
```

and then you can iterate over each table:
```
for table in tables:
    script = func.update_input_tables(create_presto_table=table['tabe'],
                                                             exesting_google_sheet_table=table['google_sheet'])
    pc.execute_batch_sql(script)
```